src/app/onboarding/components/create-new-archive/create-new-archive.component.scss

```diff
@@ -0,0 +1,130 @@
+@import '../welcome-screen/welcome-screen.component.scss';
+
+form {
+  margin: 0;
+  padding: 0;
+  max-width: 100%;
+}
+
+.two-buttons {
+  button {
+    @media only screen and (max-width: $mobile-breakpoint) {
+      width: 49%;
+    }
+    width: auto;
+    margin: 0;
+  }
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  margin: 2em 0;
+}
+
+.selected-archive {
+  & > div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 50%;
+  }
+
+  @media only screen and (max-width: $mobile-breakpoint) {
+    max-width: 100%;
+    border: 1px solid #C8C8C8;
+    box-sizing: border-box;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 8px;
+    padding: 20px;
+  }
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  max-width: 50%;
+  margin: 2em auto;
+}
+
+.name-field {
+  input {
+    max-width: 50%;
+  }
+  & > * {
+    margin: 10px;
+  }
+  display: flex;
+  flex-direction: row;
+  font-size: 1.5em;
+  justify-content: center;
+  align-items: center;
+}
+
+.archive-type-selector {
+  & > * {
+    @media only screen and (max-width: $mobile-breakpoint) {
+      width: 100%;
+    }
+    width: 30%;
+  }
+  @media only screen and (max-width: $mobile-breakpoint) {
+    flex-direction: column;
+  }
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.archive-type-name {
+  margin-bottom: 7px;
+  align-self: center;
+}
+
+.archive-type-description {
+  color: #646464;
+}
```

Comment:
I think we want to make sure to visually separate this from the name!

```suggestion
.archive-type-description {
  color: #646464;
  display: block;
}
```

Otherwise it renders all smushed together:

![Screen Shot 2022-02-24 at 12 09 38](https://user-images.githubusercontent.com/1494855/155599699-fc463928-4a24-4b81-b53e-b189cb535375.png)

Reply:
Oh, actually, now that I look at the next page which renders correctly, this might just be the same problem as the `align-self` comment below: I think they need to be in a flex container!